src/xrEngine/device.cpp

```diff
@@ -34,6 +34,7 @@ ENGINE_API BOOL g_bRendering = FALSE;
 
 BOOL g_bLoaded = FALSE;
 ref_light precache_light = 0;
+int g_dwFPSlimit = 120;
```

Comment:
Are you sure they introduced a general FPS lock though? Like I said, if we just make the main menu code more sensible we can probably fix it without enforcing a general FPS lock.